digitizer/parse.py

```diff
@@ -51,12 +53,16 @@ def prepare_isotherm_dict(form):
     data['isotherm_type'] = form.inp_isotherm_type.value
     data['measurement_type'] = form.inp_measurement_type.value
 
-    measurements = re.sub(' +', ' ', form.inp_isotherm_data.value)
-    measurements = measurements.replace(" ", ",")
-    measurements = np.genfromtxt(StringIO(measurements),
-                                 delimiter=',',
-                                 comments='#')
-    measurements = np.array(measurements, ndmin=2)  # deal with single data row
+    measurements = form.inp_isotherm_data.value  # get string from input text box
+    measurements = measurements.replace('\t', ' ')  # replace tabs with spaces
```

Comment:
Since you're already replacing tabs with spaces here, can you remove tabs as a delimiter below?

Reply:
corrected in additional commit